cmd/serve/serve.go

```diff
@@ -16,7 +18,7 @@ func NewServeCmd() *cobra.Command {
 		Use:   "serve",
 		Short: "serve",
 		RunE: func(cmd *cobra.Command, args []string) error {
-			cl, err := client.NewClient()
+			cl, err := client.NewClient(context.Background())
```

Comment:
I think you can leverage the context from the [command](https://pkg.go.dev/github.com/spf13/cobra#Command.Context)